app/views/miq_ae_customization/_dialog_sample.html.haml

```diff
@@ -64,7 +64,7 @@
                             %label{:for => "dialog_field_radio_sample_2"} Option 2
 
                           - if field.type == "DialogFieldDropDownList"
-                            = select_tag("#{field.id}", options_for_select([_("Option 1"), _("Option 2")]))
+                            = select_tag("#{field.id}", options_for_select([_("Option 1"), _("Option 2")]), {:class => 'selectpicker'})
```

Comment:
One minor thing, you don't need the curly braces around the last hash argument.

Reply:
Removed, thanks!